docs/bucket/throttling/README.md

```diff
@@ -0,0 +1,8 @@
+# Bucket Throttling Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+MinIO bucket level throttling allows for buckets to be configured with 
+
+- a bandwidth cap for both ingress and egress. Provides a way to provision bucket for customers with expected SLAs.
+- a number of requests per second. Provides a way to provision bucket for customers with expected SLAs.
```

Comment:
It presume the reqs/s is implemented elsewhere, since I don't see it in the code.

Reply:
Not yet @klauspost not going to be in this PR.

Reply:
Docs I am still working on so it's not yet updated.